### PR TITLE
Added NPC Stewart

### PR DESCRIPTION
**Pull request by Aura-Cryptic**

### CODE REVIEW

File: system/scripts/npcs/dunbarton/stewart.cs [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+				Greet();
+				Msg(Hide.Name, GetMoodString(), FavorExpression());
+				if (Player.Titles.SelectedTitle == 11002)
+					Msg("Welcome, <username/>, Guardian of Erinn.<br/>It seems like we should already start writing about your legacy<br/>along with the legendary three warriors. Haha...");
+				await Conversation();
+				break;
+
+			case "@shop":
+				Msg("I have a few items related to magic here.<br/>You can buy some if you need any.");
+				OpenShop("StweartShop");
```
Comment (Xcelled) [inline system/scripts/npcs/dunbarton/stewart.cs]: Is this misspelling a problem?

File: system/scripts/npcs/dunbarton/stewart.cs [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+				return;
+
+			case "@repair":
+				Msg("Do you want to repair your magic weapon?<br/>All magic weapons are laden with Mana, so it's impossible to physically fix them.<br/>If you fix them the way blacksmiths fix swords, then they may lose all the magic powers that come with them.");
+				Msg("Unimplemented");
```
Comment (Xcelled) [inline system/scripts/npcs/dunbarton/stewart.cs]: I thought repairs are implemented now...?

File: system/scripts/npcs/dunbarton/stewart.cs (hunk 1 above)
Comment (exectails) [inline system/scripts/npcs/dunbarton/stewart.cs]: It is, the shop script is named correctly.

File: system/scripts/npcs/dunbarton/stewart.cs (hunk 2 above)
Comment (Aura-Cryptic) [inline system/scripts/npcs/dunbarton/stewart.cs]: I'll eventually get to that and do them all at once.

Comment (exectails) [inline system/scripts/npcs/dunbarton/stewart.cs]: Yup. Just like upgrades v .

File: system/scripts/npcs/dunbarton/stewart.cs (hunk 1 above)
Comment (Aura-Cryptic) [inline system/scripts/npcs/dunbarton/stewart.cs]: Yep

**Pull request merged**